### PR TITLE
[NEW] website_sale_tax_toggle: Test more resilient to pricelists

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
ping @carlosdauden @Tardo

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: website_sale_tax_toggle/tests/test_website_sale_tax_toggle.py [hunk 1]
```diff
@@ -18,6 +18,11 @@ def setUp(self):
         })
         self.product_template.list_price = 750.00
         self.product_template.taxes_id = [(6, 0, self.tax.ids)]
+        pricelist = self.env['product.pricelist'].create({
+            'name': 'Price list for tests',
+        })
+        self.env.user.partner_id.property_product_pricelist = pricelist
+
```
Comment (carlosdauden) [inline website_sale_tax_toggle/tests/test_website_sale_tax_toggle.py:32]: Remove line
```suggestion
```

**Review by mamcode — APPROVED** [review]
LGTM, thanks.

**Comment by sergio-teruel** [comment]
Ready to be merged!!!

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**